native-code/ios/index.md

```diff
@@ -120,15 +112,52 @@ Using this, you can select the desired target and platform in the Xcode usual
 fashion and build/deploy. Note that you will need to rerun GN if you want to
 switch target platforms.
 
+### Running the tests
+
+There are several test targets in WebRTC. To run them, simply invoke them from
+the command line, for example:
+
+~~~~~ bash
+./out/mac/peerconnection_unittests
+~~~~~
+
+To run a specific test or collection of tests, pass the `--gtest_filter` argument
```

Comment:
That's not true, it can be supplied. I should probably add instructions on how to do that as well!